infrastructure/running-a-node/setup-full-node.md

```diff
@@ -17,44 +17,43 @@ Like many other protocols, A Polkadot SDK-based blockchain starts with the  _gen
 
 When a validator [seals](todo:add_link) block 1, it takes the blockchain's state at block zero. It then applies all pending changes on top of it and emits the events resulting from these changes. Later, the chain’s state at block one is used the same way to build the chain’s state at block two, and so on. Once two-thirds of the validators agree on a specific block being valid, it is finalized.
 
-#### Full Node
+=== "Full Node"
```

Comment:
Ah this is my bad. I thought the tabs would be used for the commands instead of the ## Running a Full Node, ## Running an RPC Node, and ### Running a Pruned Node sections

For informational stuff like this, I think it's better to present the information outside of tabs for increased visibility and it makes it easier for users to compare the different options, rather than toggling back and forth between tabs to understand the differences. Whereas with commands, at that point they're not really learning about the different options, they should be at a stage where they've already chosen an option and so tabs make more sense to me.

Wdyt?

Reply:
Ahhh got it, no that makes sense, let me fix that

Reply:
Done, feel free to shift content around to make it more natural if need be, thanks!